msal-dotnet-articles/contribute/build-and-test.md

```diff
@@ -1,44 +1,55 @@
 ---
 title: Building and testing MSAL.NET
+description: "How to build and test MSAL.NET on your local machine."
 ---
 
 # Building and testing MSAL.NET
 
 ## Prerequisites to build MSAL.NET
 
-Prerequisites to build MSAL.NET can be found [in our documentation](prerequisites-to-build.md).
+Prerequisites to build MSAL.NET can be found in [Prerequisites to build MSAL.NET](prerequisites-to-build.md).
 
 ## Fast build
 
-MSAL.Net supports MANY target frameworks (seven!), but most of the time contributors are only interested in one or two. To get MSAL to build for all the frameworks, you'll need a hefty Visual Studio 2019 installation plus several Windows SDKs.
+MSAL.NET supports several target frameworks, but most of the time contributors are only interested in one or two. To get MSAL to build for all frameworks, contributors will need a [hefty Visual Studio installation as well as several SDKs](prerequisites-to-build.md).
 
-To work around against this, open [this file](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/master/src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj#L3) and comment out the targets you are not interested in. Keeping the pure .NET targets and eliminating UWP and Xamarin results in a fast build and you can run all unit tests.
+To work around this requirement, open [`Microsoft.Identity.Client.csproj`](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/master/src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj) and comment out the targets you are not interested in. Keeping the pure .NET targets and eliminating UWP and Xamarin results in a fast build as well as the ability to run all unit tests.
```

Comment:
Last time I tried this didn't work any longer? can we confirm this is true?

Reply:
I think this still works - I just tried in Visual Studio 2022 and I can comment out specific sub-SKUs to get a faster build.